read_mars/__init__.py

```diff
@@ -42,3 +42,35 @@ def read_mars(filename, tree='Events', verbose=False):
     f.Close()
 
     return pd.DataFrame(events)
+
+
+def read_mars_fast(filename, tree='Events', leaves=[]):
```

Comment:
Do we need this is second function or might it be better to replace the original implementation? If it does the same and is just faster, I would replace the old code.